numpy/ma/tests/test_core.py

```diff
@@ -2581,6 +2581,14 @@ def test_no_masked_nan_warnings(self):
             # also check that allclose uses ma ufuncs, to avoid warning
             allclose(m, 0.5)
 
+    def test_masked_array_underflow(self):
+        with(np.errstate(under="raise")):
+            x = np.arange(0, 3, 0.1)
+            X = np.ma.array(x)
+            x2 = x/2.0  # <- works
+            X2 = X/2.0
+            assert(X2.all() == X.all() / 2.0)
```

Comment:
What does this assert test?

Reply:
The purpose is to make sure that the division occured fine without the multiply underflow error